HW_1/Task_1.py

```diff
@@ -0,0 +1,8 @@
+# Найдите сумму цифр трехзначного числа.
+
+print ("Введите трехзначное число, чтобы узнать сумму его цифр: ")
+n = int(input())
```

Comment:
Функцию `print` использовать необязательно. Вывод сообщения перед вводом можно осуществлять через функцию input().
```suggestion
n = int(input("Введите трехзначное число, чтобы узнать сумму его цифр: "))
```